code/controllers/subsystem/mapping.dm

```diff
@@ -19,6 +19,11 @@ SUBSYSTEM_DEF(mapping)
 	var/list/shelter_templates = list()
 
 	var/loading_ruins = FALSE
+	var/list/turf/unused_turfs = list()				//Not actually unused turfs they're unused but reserved for use for whatever requests them. "[zlevel_of_turf]" = list(turfs)
+	var/list/datum/turf_reservations = list()		//list of turf reservations
+	var/list/turf/used_turfs = list()				//list of turf = datum/turf_reservations
```

Comment:
Wtf this list contains the datum AND turfs?! KEVINZZ!!!!!

Reply:
associated I'd assume.